src/index.js

```diff
@@ -1,17 +1,18 @@
 const express = require('express');
 const app = express();
-const path = require('path');
 const events = require('events');
+const debug = require('debug');
+const path  = require('path');
+const defaultValues = require('../defaultValues.js');
 
 
-app.get('/', (req, res) => {
-    const htmlPath = path.join(__dirname, 'index.html');
+const log = debug('mylib:messages');
 
-    res.sendFile(htmlPath);
-});
+
+app.use(express.static(path.join(path.parse(__dirname).dir, 'resources')));
```

Comment:
Good enough, but you can simplify it with  `path.join(__dirname, '../resources')`